csharp/src/Microsoft.ML.OnnxRuntime/NativeMethods.shared.cs

```diff
@@ -368,6 +368,88 @@ public struct OrtApi
         public IntPtr EpDevice_Device;
         public IntPtr GetEpApi;
         public IntPtr GetTensorSizeInBytes;
+
```

Comment:
It looks like this file has yet to be updated with all the other APIs that have been added since 1.22... I tried adding them (ended up with some extra whitespace) since the comment further up in the file said these need to be in sync. Feedback welcome on whether I should have done something else here.

Reply:
Yes, just keep adding `IntPtr` with names there, you do not need initialize them in your particular PR, but the layout needs to be preserved. And yes, some of them has already been added since 1.22.